.github/workflows/container-tests.yml

```diff
@@ -41,113 +41,32 @@ jobs:
         run: |
           _result=${{ steps.check.outputs.result }}
           if [ "$_result" == failure ]; then
-            echo "Distgen-generated files are not regenerated properly."
-            echo "Please regenerate them with:"
-            echo "'make clean-versions'"
-            echo "'make generate-all'"
+            echo "::error::Distgen-generated files are not regenerated properly."
```

Comment:
This part is missing in `.github/workflows/openshift-tests.yml`

Reply:
Yes, it was intended. The same is in python and varnish

Point is, that we want only one status line for distgen check. Not one from container tests and one from OCP tests.
Thus the status is being created only from container tests.

I realize, that this is not ideal. Do you have any better idea?

Reply:
What about if we schedule tests only for "OpenShift"? Then dist-gen check is skipped.

Reply:
I don't see any problem with duplication. distgen report should be checked always.

Reply:
But what if the content is not regenerated properly? Then we will be testing incorrect source codes.

Reply:
Oh, I see, there is a check in OCP tests too, but does not produce status.

Reply:
Please note that 'distgen check for OCP tests feature' got LGTM when it was implemented[1]. 
Also the distgen check is not connected to this PR.

If you want to change the functionality I would suggest discussing this on the container meeting and then maybe creating and issue or PR.


[1] https://github.com/sclorg/s2i-python-container/pull/558

Reply:
In case of dist-gen check failed, it would be nice to have a reason, but let's not block this pull request. It is a nice to have.